tests/test_style.py

```diff
@@ -38,6 +38,12 @@ def test_astyle():
     assert ret == 'void main() {}'
     assert styler.default_extensions()
 
+def test_eslint():
```

Comment:
Decorate this so it we can run other tests without node or eslint installed. Autopep8 doesn't have it because it is pip installed as part of zazu.
```
@pytest.mark.skipif(not distutils.spawn.find_executable('eslint'),
                    reason="requires eslint")
def test_eslint():
```